trigger/src/main/scala/core/scheduler/utilities/Configs.scala

```diff
@@ -0,0 +1,67 @@
+package core.scheduler.utilities
+
+import java.util
+import java.util.{Collections, Properties}
+
+import com.typesafe.config.{Config, ConfigFactory}
+
+private object Configs {
+  val conf: Config = ConfigFactory.load()
+}
+
+object KafkaConfig {
+  def getConsumerProperties: Properties = {
+    val properties = new Properties()
+    properties.put("bootstrap.servers", Configs.conf.getString("kafkaSource.bootstrap.servers"))
+    properties.put("group.id", Configs.conf.getString("kafkaSource.group.id"))
+    properties.put("key.deserializer", Configs.conf.getString("kafkaSource.key.deserializer"))
+    properties.put("value.deserializer", Configs.conf.getString("kafkaSource.value.deserializer"))
+    properties.put("max.poll.records", Configs.conf.getString("kafkaSource.max.poll.records"))
+    properties.put("enable.auto.commit", "false")
+    properties.put("auto.offset.reset", "latest")
+    properties
+  }
+
+  def getNotificationTopics: util.List[String] =
+    Collections.singletonList(Configs.conf.getString("kafkaSource.notificationTopic"))
```

Comment:
Maybe extract the literal into a companion object?

Reply:
Actually this is the place where It has been extracted. This place is only one place where literal is used.  Only methods are used in other parts no string literals.